### DISCUSSION TITLE
Azure Functionsを利用してCosmosDBに接続したい

### DISCUSSION THREAD

**Discussion by ksera332** [1ft-seabass/ms-azure-hackathon-league-2021#47] [Q&A] [answered]
https://docs.microsoft.com/ja-jp/azure/azure-functions/functions-bindings-cosmosdb-v2-input?tabs=javascript
上記URLを参考にAzure Functionsを利用してCosmosDB(Mongo API)からデータのfindを試みているが、うまくいかない。
DBはすべてのIPからのアクセスを許可しており、ユーザーには所有者のロールを付与している。

**Comment by papeMK2** [comment, marked as answer]
## アプリケーション設定の方法

Function App のアプリケーション設定は、`環境名.settings.json` に記述します。
ローカル開発の場合は、`local.settings.json` など

下記の例のように、Values 配下に入れていきます。

``` json
{
  "IsEncrypted": false,
  "Values": {
    "AzureWebJobsStorage": "",
    "FUNCTIONS_WORKER_RUNTIME": "python",
  }
}
```

※ 接続文字列は、コミットしないように  `.gitignore` に記載しましょう。
